### ISSUE TITLE
Try the new API to check for problems

### ISSUE THREAD

**Issue by tomahock** [FogosPT/fogosmobile#156]
Change API endpoint to api-dev.fogos.pt and check for problems.

**Comment by jcalado** [comment]
The dev API sometimes returns empty strings, and other times returns nulls.

Eg: https://api-dev.fogos.pt/new/fires
```json
extra: "",
kml: null
```

Also the production API returns 'text/html; charset=UTF-8' and the dev API returns 'application/json'.
I believe this is breaking the mobile app. Not sure what's the best course of action here.
Maybe the mobile app code should be change to deal with the 'application/json'.

**Comment by jcalado** [comment]
**Fire ID:** 2021080032936

**Dev**: https://api-dev.fogos.pt/fires?id=2021080032936
```json
"created": "2021-08-04T12:52:05.104000Z",
"dateTime": "2021-08-04T12:46:00.000000Z",
```
***

**Production**: https://api-lb.fogos.pt/fires?id=2021080032936
```json
"created":{"sec":1628081522,"usec":294000}
"dateTime":{"sec":1628081160,"usec":0}
```
Should the api match production, or should the app adapt to the new API?

***


Also, this is inconsistent with https://api-dev.fogos.pt/new/fires, which for the dev API has:

```json
"dateTime": {
    "sec": 1628081160
},
"created": {
    "sec": 1628081525
},
```

**Comment by jcalado** [comment]
https://github.com/FogosPT/fogosmobile/pull/168 in progress

**Comment by OldMetalmind** [comment]
This PR can be closed since #168 has been successfully merged.
